.ci/pipelines/release_zeebe.groovy

```diff
@@ -146,11 +146,15 @@ spec:
         }
 
         stage('Docker Image') {
+            environment {
+                // retrieve the git commit hash from the checked out tag of the release
+                REVISION = sh(returnStdout: true, script: "git --git-dir target/checkout/.git log -n 1 --pretty=format:'%h'").trim()
```

Comment:
I just modified the command for retrieving the hash as documented here https://github.com/jenkinsci/pipeline-examples/blob/master/pipeline-examples/gitcommit/gitcommit.groovy 
to point to the directory where maven checks out the tag
this is where maven checks out the actual release tag once available, see https://maven.apache.org/maven-release/maven-release-plugin/perform-mojo.html#workingDirectory

Reply:
:question: Why we have to specify the git dir?

Reply:
The main workdir will already contain next dev version commits that maven creates when the release is finished.

Reply:
Would be great to add this as a comment

Reply:
that was the intention of
```
// retrieve the git commit hash from the checked out tag of the release
``` 
😅 

Can rephrase/expand it though.

Reply:
wdyt about:
```
                // Retrieve the git commit hash from the checked out tag of the release
                // as when maven performs the perform-release step it will checkout the created tag
                // to the workingDirectory that defaults to target/checkout, see
                // https://maven.apache.org/maven-release/maven-release-plugin/examples/perform-release.html
                // The runners main workdir will already contain next release commits that
                // maven creates once the release is finished, thus the rev would not match the release.
                ```
                @Zelldon ?

Reply:
This is perfect thanks

Reply:
done